ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/useNodeCves.ts

```diff
@@ -0,0 +1,79 @@
+import { gql, useQuery } from '@apollo/client';
+import { QuerySearchFilter } from '../../types';
+import { getRegexScopedQueryString } from '../../utils/searchUtils';
+
+const cvesListQuery = gql`
+    query getNodeCVEs($query: String, $pagination: Pagination) {
+        nodeCVEs(query: $query, pagination: $pagination) {
+            cve
+            nodeCountBySeverity {
+                critical {
+                    total
+                }
+                important {
+                    total
+                }
+                moderate {
+                    total
+                }
+                low {
+                    total
+                }
+            }
+            topCVSS
+            affectedNodeCount
+            firstDiscoveredInSystem
+            distroTuples {
+                summary
+                operatingSystem
+                cvss
+                scoreVersion
+            }
+        }
+    }
+`;
+
+// TODO Need to verify these types with the BE implementation
+export type NodeCVE = {
+    cve: string;
+    nodeCountBySeverity: {
+        critical: { total: number };
+        important: { total: number };
+        moderate: { total: number };
+        low: { total: number };
+    };
+    topCVSS: number;
+    affectedNodeCount: number;
+    firstDiscoveredInSystem: string;
+    distroTuples: {
+        summary: string;
+        operatingSystem: string;
+        cvss: number;
+        scoreVersion: string;
+    }[];
+};
+
+export default function useNodeCves(
```

Comment:
An observation, not necessarily a change request:
* `getNodeCVEs`
* `NodeCVE`
* `useNodeCve`

When CVE is in the middle of a name, camelCase suggests Cve, but at the end seems like judgment call.

Reply:
Yeah this is something that seems to be consistently inconsistent across the product, at least in the UI code. Might be worth bringing up at a team meeting and deciding if we can standardize on how to handle acronyms for files, functions, etc.